sdk/core/core-http/README.md

```diff
@@ -23,14 +23,26 @@ Examples can be found in the `samples` folder.
 
 ## Next steps
 
+First obtain an access token to the management service used in the sample code
+
+One easy way to get the token using Azure CLI (https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
+
+1. `az login` using the above subscription
+2. `az account set -s <subscription id>`
+3. `az account get-access-token --resource=https://management.azure.com`
+
 ### Node.js
 
-- Set the subscriptionId and token
-- Run `node samples/node-sample.js`
+- Set the subscriptionId and token environment variables
+- Change directory to samples folder `cd samples`
+- Compile the TypeScript code `tsc node-sample.ts`
```

Comment:
what about installing TypeScript globally first? Otherwise this would fail. Maybe `npx ts-node` is another option?

Reply:
There's already an instruction to install typescript globally in earlier Requirement section.